src/arm/linux/aarch64-isa.c

```diff
@@ -151,4 +151,25 @@ void cpuinfo_arm64_linux_decode_isa_from_proc_cpuinfo(
 	if (features & CPUINFO_ARM_LINUX_FEATURE_ASIMDFHM) {
 		isa->fhm = true;
 	}
+
+#if defined(__linux__)
+#include <sys/prctl.h>
```

Comment:
Adding `#include` directive in the body of the function feels very fragile, can you please move it to the top level of the file?

Reply:
Done